src/tools/map_extractor/System.cpp

```diff
@@ -1292,6 +1292,31 @@ void ExtractGameTables()
     printf("Extracted %u files\n\n", count);
 }
 
+
+int GetLocale() {
+
+	int i = 0;
+	int result = -1;
+
+	if (!((CONF_Locale && !(CONF_Locale & (1 << i))) && (i == LOCALE_none))){
```

Comment:
local variable i is assigned to a constant value so "1 << i" can be replaced with "1" and "i == LOCALE_none" can be removed (assuming LOCALE_none is defined as = 0)